tests/MulticastUDPClient.py

```
@@ -240,11 +240,11 @@
 
 		"""
 		# skipcq: TCV-002
-		if str("grp_addr") in kwargs:
+		if str("grp_addr") in kwargs:  # pragma: no branch
```

Ruff (0.8.2)

Unnecessary `str` call (rewrite as a literal)

Replace with string literal

(UP018)